[TASK] Add issue templates (#706)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: .github/ISSUE_TEMPLATE/feature_request.md

---
name: 🚀 Feature request
about: Suggest an idea for this documentation (and may want to implement it 🙂)

---

> ### Is your feature request related to a problem? Please describe.

Author: ```suggestion
### Is your feature request related to an issue? Please describe.
```

> A clear and concise description about your idea, possibly related to some problem.

Author: ```suggestion
A clear and concise description about your idea.
```

> A clear and concise description of what the problem is. Ex. I have an issue when [...]

Reviewer: ```suggestion
A clear and concise description about your idea, possibly related to some problem.
```
